static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-times-of-day.js

```diff
@@ -0,0 +1,27 @@
+
+
+export const signInGateTimesOfDay = {
+    id: 'SignInGateTimesOfDay',
+    start: '2023-10-23',
+    expiry: '2023-11-23',
+    author: 'Lindsey Dew',
+    description:
+        'Show sign in gate more frequently to users in the morning',
+    audience: 0.5,
+    audienceOffset: 0.4,
```

Comment:
I think the main gate AB definition needs to be amended to match in frontend too.